client/js/components/map/admin/AdminLayerList.vue

```diff
@@ -130,88 +130,89 @@
         {{ Translator.trans('no.data') }}
       </div>
 
-      <h3 class="u-mt">
-        {{ Translator.trans('map.bases') }}
-      </h3>
-      <!-- List-Head -->
-      <div class="color--grey u-mb-0_25 u-mt-0_5 u-mr-0_5">
-        <div class="c-at-item__row-icon layout__item u-pl-0">
-          <!-- DragHandler -->
-        </div><!--
-     --><div class="layout--flush layout__item c-at-item__row">
-            <div class="layout__item w-10/12 u-pl-0_5">
-              {{ Translator.trans('description') }}
-            </div><!--
-         --><div class="layout__item w-1/12 text-right">
-            <i
-              class="fa fa-eye u-mr-0_5"
-              v-tooltip="Translator.trans('explanation.gislayer.visibility')" />
-            </div><!--
-         --><div class="layout__item w-1/12 text-right">
-                {{ Translator.trans('edit') }}
-            </div>
+      <div v-if="hasPermission('feature_map_layer_visibility')">
+        <h3 class="u-mt">
+          {{ Translator.trans('map.bases') }}
+        </h3>
+        <!-- List-Head -->
+        <div class="color--grey u-mb-0_25 u-mt-0_5 u-mr-0_5">
+          <div class="c-at-item__row-icon layout__item u-pl-0">
+            <!-- DragHandler -->
+          </div><!--
+     -  -><div class="layout--flush layout__item c-at-item__row">
```

Comment:
If You change the blanks here, it will destroy the comment and with that the template
```suggestion
       --><div class="layout--flush layout__item c-at-item__row">
```

Reply:
done in [a204fe9](https://github.com/demos-europe/demosplan-core/pull/3345/commits/a204fe925ab89ec597f0409e2f18fbd98b750bc6)